File: ympa_fsharp_giraffe_server/src/model/CampaignSettingsDeliveryDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.CampaignSettingsScheduleDTO

module CampaignSettingsDeliveryDTO =

  //#region CampaignSettingsDeliveryDTO


  type CampaignSettingsDeliveryDTO = {
    Schedule : CampaignSettingsScheduleDTO;
  }
  //#endregion

File: app/model/CampaignSettingsDeliveryDTO.scala
package model

import play.api.libs.json._

/**
  * Информация о доставке в своем регионе магазина.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2024-09-09T22:33:09.055496365Z[Etc/UTC]", comments = "Generator version: 7.8.0")
case class CampaignSettingsDeliveryDTO(
  schedule: Option[CampaignSettingsScheduleDTO]
)

object CampaignSettingsDeliveryDTO {
  implicit lazy val campaignSettingsDeliveryDTOJsonFormat: Format[CampaignSettingsDeliveryDTO] = Json.format[CampaignSettingsDeliveryDTO]
}

